### DISCUSSION TITLE
Select component with search and create capability

### DISCUSSION THREAD

**Discussion by woernerm** [reflex-dev/reflex#3103] [Show and tell]
Hi!
I created a select component which allows the user to search for options and create new ones. It is called [Dynoselect](https://github.com/woernerm/dynoselect):

![demo](https://github.com/reflex-dev/reflex/assets/61889355/f52ebe5b-728b-4761-819c-53e97ebc2919)

You can install it using:
```py
pip install reflex-dynoselect
```
Enjoy! :)

**Comment by Alek99** [comment]
This is awesome!!

**Reply by woernerm** [reply]
Thanks. ;)

**Reply by CohPhol** [reply]
Hi, is there any update for the dynamic data?
